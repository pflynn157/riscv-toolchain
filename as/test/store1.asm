sw x1, 20(x2)
sb x1, 20(x3)
sh x2, 20(x4)
