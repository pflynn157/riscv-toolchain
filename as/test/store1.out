00112a23
00118a23
00221a23
